Sofa/GUI/CMakeLists.txt

```
@@ -4,12 +4,12 @@ add_subdirectory(Component)
 add_subdirectory(Common)
 add_subdirectory(Batch)
 
-option(SOFAGUI_QT "Compile the Qt GUI" ON)
-if(SOFAGUI_QT)
+option(SOFA_GUI_ENABLE_QT "Compile the Qt GUI" ON)
+if(SOFA_GUI_ENABLE_QT)
     add_subdirectory(Qt)
 endif()
 
-option(SOFAGUI_HEADLESS_RECORDER "Compile the headless recorder (Linux only)" OFF)
-if(SOFAGUI_HEADLESS_RECORDER)
+option(SOFA_GUI_ENABLE_HEADLESS_RECORDER "Compile the headless recorder (Linux only)" OFF)
+if(SOFA_GUI_ENABLE_HEADLESS_RECORDER)
```

```suggestion
option(SOFA_GUI_ENABLE_HEADLESSRECORDER "Compile the headless recorder (Linux only)" OFF)
if(SOFA_GUI_ENABLE_HEADLESSRECORDER)
```
Looks more consistent to me. May need to be propagated somewhere else...

---

seems not (after grep'ping, it is the only two occurences)